contract_payment_mode/hooks.py

```diff
@@ -10,14 +10,14 @@
 
 def post_init_hook(cr, registry):
     """Copy payment mode from partner to the new field at contract."""
-    with api.Environment.manage():
-        env = api.Environment(cr, SUPERUSER_ID, {})
-        m_contract = env["contract.contract"]
-        contracts = m_contract.search([("payment_mode_id", "=", False)])
-        if contracts:
-            _logger.info("Setting payment mode: %d contracts" % len(contracts))
-        for contract in contracts:
-            payment_mode = contract.partner_id.customer_payment_mode_id
-            if payment_mode:
-                contract.payment_mode_id = payment_mode.id
-        _logger.info("Setting payment mode: Done")
+    # with api.Environment.manage():
```

Comment:
Why do you remove the context manager  ?

Reply:
Ok, deprecated.

@palkesh-aktivsoftware Could you just remove the unneeded comment ?

Reply:
> 

Sure, I made this changes. Please review it.